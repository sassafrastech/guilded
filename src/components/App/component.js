import React from 'react';
import {
  createMuiTheme,
  ThemeProvider,
  CssBaseline,
  makeStyles,
} from '@material-ui/core';

import Drawer from '../Drawer/component';
import Hero from '../Hero/component';
import Services from '../Services/component';

import '@fontsource/roboto';
import appTheme from '../../theme';

const muiTheme = createMuiTheme(appTheme);

const useStyles = makeStyles({
  container: {
    display: 'flex',
    width: '100%',
  },
  mainContainer: {
    flex: 1,
  },
});

const App = () => {
  const classes = useStyles();
  const [isDrawerOpen, setDrawerOpen] = React.useState(true);

  const handleDrawerOpen = () => {
    setDrawerOpen(true);
  };

  const handleDrawerClose = () => {
    setDrawerOpen(false);
  };

  return (
    <ThemeProvider theme={muiTheme}>
      <CssBaseline />
      <div className={classes.container}>
        <Drawer
          isOpen={isDrawerOpen}
          handleOpen={handleDrawerOpen}
          handleClose={handleDrawerClose}
        />
        <main className={classes.mainContainer}>
          <Hero />
          <Services />
        </main>
      </div>
    </ThemeProvider>
  );
};

export default App;
