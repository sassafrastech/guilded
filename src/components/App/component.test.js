import React from 'react';
import renderer from 'react-test-renderer';

import App from './component';

it('renders', () => {
  const tree = renderer.create(<App />);
  expect(tree.toJSON()).toMatchSnapshot();
});
