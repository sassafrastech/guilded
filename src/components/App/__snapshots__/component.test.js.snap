// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="makeStyles-container-1"
>
  <Drawer
    handleClose={[Function]}
    handleOpen={[Function]}
    isOpen={true}
  />
  <main
    className="makeStyles-mainContainer-2"
  >
    <Hero />
    <Services />
  </main>
</div>
`;
